# ru.po
# LIBPQ Translated Messages into the Russian Language (UTF-8)
#
# Copyright (c) 2001-2004 Serguei A. Mokhov, mokhov@cs.concordia.ca
# Copyright (c) 2012 Alexander Lakhin <exclusion@gmail.com>
# Distributed under the same licensing terms as PostgreSQL itself.
#
# pgtranslation Id: libpq.po,v 1.4 2010/12/07 21:22:21 petere Exp $
#
# ChangeLog:
#
#   - April 2, 2012: Bug fixes. Alexander Lakhin <exclusion@gmail.com>.
#   - February 22, 2012: Backporting 9.1 translation to 8.4. Alexander Lakhin <exclusion@gmail.com>.
#   - February 17, 2012: complete translation for 9.1. Alexander Lakhin <exclusion@gmail.com>.
#   - November, 2010: Updates for PostgreSQL 9 by Andrey Sudnik <sudnikand@yandex.ru>.
#   - January, 2005: Corrections and improvements by Oleg Bartunov <oleg@sai.msu.su>.
#   - March 27 - November 1, 2004: Updates for 8.0; <mokhov@cs.concordia.ca>.
#   - July 24 - October 5, 2003: Updates for 7.4.*; <mokhov@cs.concordia.ca>.
#   - Januray 2, 2002: Completed SSL translation, Serguei A. Mokhov <mokhov@cs.concordia.ca>.
#   - October 12, 2002: Post-7.3beta2 fuzzy translation fixes, Serguei A. Mokhov <mokhov@cs.concordia.ca>.
#   - September 7, 2002: Complete post-7.3beta1 Translation, Serguei A. Mokhov <mokhov@cs.concordia.ca>.
#   - August 2001 - August 2002: Initial translation and maintenance, Serguei A. Mokhov <mokhov@cs.concordia.ca>
msgid ""
msgstr ""
"Project-Id-Version: PostgreSQL 9.1\n"
"Report-Msgid-Bugs-To: pgsql-bugs@postgresql.org\n"
"POT-Creation-Date: 2013-08-18 21:09+0000\n"
"PO-Revision-Date: 2013-08-19 09:20+0400\n"
"Last-Translator: Alexander Lakhin <exclusion@gmail.com>\n"
"Language-Team: Russian <pgtranslation-translators@pgfoundry.org>\n"
"Language: ru\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Poedit-Language: Russian\n"
"X-Poedit-Country: RUSSIAN FEDERATION\n"
"X-Poedit-SourceCharset: utf-8\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
"X-Generator: Lokalize 1.5\n"

#: fe-auth.c:210 fe-auth.c:429 fe-auth.c:656
msgid "host name must be specified\n"
msgstr "требуется указать имя сервера\n"

#: fe-auth.c:240
#, c-format
msgid "could not set socket to blocking mode: %s\n"
msgstr "не удалось перевести сокет в блокирующий режим: %s\n"

#: fe-auth.c:258 fe-auth.c:262
#, c-format
msgid "Kerberos 5 authentication rejected: %*s\n"
msgstr "аутентификация Kerberos 5 не пройдена: %*s\n"

#: fe-auth.c:288
#, c-format
msgid "could not restore non-blocking mode on socket: %s\n"
msgstr "не удалось вернуть сокет в неблокирующий режим: %s\n"

#: fe-auth.c:400
msgid "GSSAPI continuation error"
msgstr "ошибка продолжения в GSSAPI"

#: fe-auth.c:436
msgid "duplicate GSS authentication request\n"
msgstr "повторный запрос аутентификации GSS\n"

#: fe-auth.c:456
msgid "GSSAPI name import error"
msgstr "ошибка импорта имени в GSSAPI"

#: fe-auth.c:542
msgid "SSPI continuation error"
msgstr "ошибка продолжения в SSPI"

#: fe-auth.c:553 fe-auth.c:627 fe-auth.c:662 fe-auth.c:757 fe-connect.c:1971
#: fe-connect.c:3380 fe-connect.c:3598 fe-connect.c:4019 fe-connect.c:4028
#: fe-connect.c:4165 fe-connect.c:4211 fe-connect.c:4229 fe-connect.c:4308
#: fe-connect.c:4378 fe-connect.c:4424 fe-connect.c:4442 fe-exec.c:3121
#: fe-exec.c:3286 fe-lobj.c:711 fe-protocol2.c:1092 fe-protocol3.c:1433
#: fe-secure.c:789 fe-secure.c:1189
msgid "out of memory\n"
msgstr "нехватка памяти\n"

#: fe-auth.c:642
msgid "could not acquire SSPI credentials"
msgstr "не удалось получить удостоверение SSPI"

#: fe-auth.c:733
msgid "SCM_CRED authentication method not supported\n"
msgstr "аутентификация SCM_CRED не поддерживается\n"

#: fe-auth.c:807
msgid "Kerberos 4 authentication not supported\n"
msgstr "аутентификация Kerberos 4 не поддерживается\n"

#: fe-auth.c:823
msgid "Kerberos 5 authentication not supported\n"
msgstr "аутентификация Kerberos 5 не поддерживается\n"

#: fe-auth.c:895
msgid "GSSAPI authentication not supported\n"
msgstr "аутентификация через GSSAPI не поддерживается\n"

#: fe-auth.c:927
msgid "SSPI authentication not supported\n"
msgstr "аутентификация через SSPI не поддерживается\n"

#: fe-auth.c:935
msgid "Crypt authentication not supported\n"
msgstr "аутентификация Crypt не поддерживается\n"

#: fe-auth.c:962
#, c-format
msgid "authentication method %u not supported\n"
msgstr "метод аутентификации %u не поддерживается\n"

#: fe-connect.c:758
#, c-format
msgid "invalid sslmode value: \"%s\"\n"
msgstr "неверное значение sslmode: \"%s\"\n"

#: fe-connect.c:779
#, c-format
msgid "sslmode value \"%s\" invalid when SSL support is not compiled in\n"
msgstr "значение sslmode \"%s\" недопустимо для сборки без поддержки SSL\n"

#: fe-connect.c:972
#, c-format
msgid "could not set socket to TCP no delay mode: %s\n"
msgstr "не удалось перевести сокет в режим TCP-передачи без задержки: %s\n"

#: fe-connect.c:1002
#, c-format
msgid ""
"could not connect to server: %s\n"
"\tIs the server running locally and accepting\n"
"\tconnections on Unix domain socket \"%s\"?\n"
msgstr ""
"не удалось подключиться к серверу: %s\n"
"\tОн действительно работает локально и принимает\n"
"\tсоединения через доменный сокет \"%s\"?\n"

#: fe-connect.c:1057
#, c-format
msgid ""
"could not connect to server: %s\n"
"\tIs the server running on host \"%s\" (%s) and accepting\n"
"\tTCP/IP connections on port %s?\n"
msgstr ""
"не удалось подключиться к серверу: %s\n"
"\tОн действительно работает по адресу \"%s\" (%s)\n"
"\t и принимает TCP-соединения (порт %s)?\n"

#: fe-connect.c:1066
#, c-format
msgid ""
"could not connect to server: %s\n"
"\tIs the server running on host \"%s\" and accepting\n"
"\tTCP/IP connections on port %s?\n"
msgstr ""
"не удалось подключиться к серверу: %s\n"
"\tОн действительно работает по адресу \"%s\"\n"
"\t и принимает TCP-соединения (порт %s)?\n"

#: fe-connect.c:1117
#, c-format
msgid "setsockopt(TCP_KEEPIDLE) failed: %s\n"
msgstr "ошибка в setsockopt(TCP_KEEPIDLE): %s\n"

#: fe-connect.c:1130
#, c-format
msgid "setsockopt(TCP_KEEPALIVE) failed: %s\n"
msgstr "ошибка в setsockopt(TCP_KEEPALIVE): %s\n"

#: fe-connect.c:1162
#, c-format
msgid "setsockopt(TCP_KEEPINTVL) failed: %s\n"
msgstr "ошибка в setsockopt(TCP_KEEPINTVL): %s\n"

#: fe-connect.c:1194
#, c-format
msgid "setsockopt(TCP_KEEPCNT) failed: %s\n"
msgstr "ошибка в setsockopt(TCP_KEEPCNT): %s\n"

#: fe-connect.c:1242
#, c-format
msgid "WSAIoctl(SIO_KEEPALIVE_VALS) failed: %ui\n"
msgstr "ошибка в WSAIoctl(SIO_KEEPALIVE_VALS): %ui\n"

#: fe-connect.c:1294
#, c-format
msgid "invalid port number: \"%s\"\n"
msgstr "неверный номер порта: \"%s\"\n"

#: fe-connect.c:1327
#, c-format
msgid "Unix-domain socket path \"%s\" is too long (maximum %d bytes)\n"
msgstr "длина пути доменного сокета \"%s\" превышает предел (%d байт)\n"

#: fe-connect.c:1346
#, c-format
msgid "could not translate host name \"%s\" to address: %s\n"
msgstr "преобразовать имя \"%s\" в адрес не удалось: %s\n"

#: fe-connect.c:1350
#, c-format
msgid "could not translate Unix-domain socket path \"%s\" to address: %s\n"
msgstr ""
"преобразовать путь к доменному сокету UNIX \"%s\" в адрес не удалось: %s\n"

#: fe-connect.c:1560
msgid "invalid connection state, probably indicative of memory corruption\n"
msgstr "неверное состояние соединения - возможно разрушение памяти\n"

#: fe-connect.c:1601
#, c-format
msgid "could not create socket: %s\n"
msgstr "не удалось создать сокет: %s\n"

#: fe-connect.c:1624
#, c-format
msgid "could not set socket to non-blocking mode: %s\n"
msgstr "не удалось перевести сокет в неблокирующий режим: %s\n"

#: fe-connect.c:1636
#, c-format
msgid "could not set socket to close-on-exec mode: %s\n"
msgstr ""
"не удалось перевести сокет в режим закрытия при выполнении (close-on-exec): "
"%s\n"

#: fe-connect.c:1656
msgid "keepalives parameter must be an integer\n"
msgstr "параметр keepalives должен быть целым числом\n"

#: fe-connect.c:1669
#, c-format
msgid "setsockopt(SO_KEEPALIVE) failed: %s\n"
msgstr "ошибка в setsockopt(SO_KEEPALIVE): %s\n"

#: fe-connect.c:1811
#, c-format
msgid "could not get socket error status: %s\n"
msgstr "не удалось получить статус ошибки сокета: %s\n"

#: fe-connect.c:1849
#, c-format
msgid "could not get client address from socket: %s\n"
msgstr "не удалось получить адрес клиента из сокета: %s\n"

#: fe-connect.c:1890
msgid "requirepeer parameter is not supported on this platform\n"
msgstr "параметр requirepeer не поддерживается в этой ОС\n"

#: fe-connect.c:1893
#, c-format
msgid "could not get peer credentials: %s\n"
msgstr "не удалось получить учётные данные сервера: %s\n"

#: fe-connect.c:1903
#, c-format
msgid "local user with ID %d does not exist\n"
msgstr "локальный пользователь с ID %d не существует\n"

#: fe-connect.c:1911
#, c-format
msgid "requirepeer specifies \"%s\", but actual peer user name is \"%s\"\n"
msgstr ""
"requirepeer допускает подключение только к \"%s\", но сервер работает под "
"именем \"%s\"\n"

#: fe-connect.c:1945
#, c-format
msgid "could not send SSL negotiation packet: %s\n"
msgstr "не удалось отправить пакет согласования SSL: %s\n"

#: fe-connect.c:1984
#, c-format
msgid "could not send startup packet: %s\n"
msgstr "не удалось отправить стартовый пакет: %s\n"

#: fe-connect.c:2054
msgid "server does not support SSL, but SSL was required\n"
msgstr "затребовано подключение через SSL, но сервер не поддерживает SSL\n"

#: fe-connect.c:2080
#, c-format
msgid "received invalid response to SSL negotiation: %c\n"
msgstr "получен неверный ответ на согласование SSL: %c\n"

#: fe-connect.c:2159 fe-connect.c:2192
#, c-format
msgid "expected authentication request from server, but received %c\n"
msgstr "ожидался запрос аутентификации от сервера, но получено: %c\n"

#: fe-connect.c:2373
#, c-format
msgid "out of memory allocating GSSAPI buffer (%i)"
msgstr "недостаточно памяти для буфера GSSAPI (%i)"

#: fe-connect.c:2458
msgid "unexpected message from server during startup\n"
msgstr "неожиданное сообщение от сервера в начале работы\n"

#: fe-connect.c:2557
#, c-format
msgid "invalid connection state %d, probably indicative of memory corruption\n"
msgstr "неверное состояние соединения %d - возможно разрушение памяти\n"

#: fe-connect.c:2988 fe-connect.c:3048
#, c-format
msgid "PGEventProc \"%s\" failed during PGEVT_CONNRESET event\n"
msgstr "ошибка в PGEventProc \"%s\" при обработке события PGEVT_CONNRESET\n"

#: fe-connect.c:3393
#, c-format
msgid "invalid LDAP URL \"%s\": scheme must be ldap://\n"
msgstr "некорректный адрес LDAP \"%s\": схема должна быть ldap://\n"

#: fe-connect.c:3408
#, c-format
msgid "invalid LDAP URL \"%s\": missing distinguished name\n"
msgstr "некорректный адрес LDAP \"%s\": отсутствует уникальное имя\n"

#: fe-connect.c:3419 fe-connect.c:3472
#, c-format
msgid "invalid LDAP URL \"%s\": must have exactly one attribute\n"
msgstr "некорректный адрес LDAP \"%s\": должен быть только один атрибут\n"

#: fe-connect.c:3429 fe-connect.c:3486
#, c-format
msgid "invalid LDAP URL \"%s\": must have search scope (base/one/sub)\n"
msgstr ""
"некорректный адрес LDAP \"%s\": не указана область поиска (base/one/sub)\n"

#: fe-connect.c:3440
#, c-format
msgid "invalid LDAP URL \"%s\": no filter\n"
msgstr "некорректный адрес LDAP \"%s\": нет фильтра\n"

#: fe-connect.c:3461
#, c-format
msgid "invalid LDAP URL \"%s\": invalid port number\n"
msgstr "некорректный адрес LDAP \"%s\": неверный номер порта\n"

#: fe-connect.c:3495
msgid "could not create LDAP structure\n"
msgstr "не удалось создать структуру LDAP\n"

#: fe-connect.c:3537
#, c-format
msgid "lookup on LDAP server failed: %s\n"
msgstr "ошибка поиска на сервере LDAP: %s\n"

#: fe-connect.c:3548
msgid "more than one entry found on LDAP lookup\n"
msgstr "при поиске LDAP найдено более одного вхождения\n"

#: fe-connect.c:3549 fe-connect.c:3561
msgid "no entry found on LDAP lookup\n"
msgstr "при поиске LDAP не найдено ничего\n"

#: fe-connect.c:3572 fe-connect.c:3585
msgid "attribute has no values on LDAP lookup\n"
msgstr "атрибут не содержит значений при поиске LDAP\n"

#: fe-connect.c:3637 fe-connect.c:3656 fe-connect.c:4067
#, c-format
msgid "missing \"=\" after \"%s\" in connection info string\n"
msgstr "в строке соединения нет \"=\" после \"%s\"\n"

#: fe-connect.c:3720 fe-connect.c:4149 fe-connect.c:4333
#, c-format
msgid "invalid connection option \"%s\"\n"
msgstr "неверный параметр соединения \"%s\"\n"

#: fe-connect.c:3736 fe-connect.c:4116
msgid "unterminated quoted string in connection info string\n"
msgstr "в строке соединения не хватает закрывающей кавычки\n"

#: fe-connect.c:3775
msgid "could not get home directory to locate service definition file"
msgstr ""
"не удалось получить домашний каталог для загрузки файла определений служб"

#: fe-connect.c:3808
#, c-format
msgid "definition of service \"%s\" not found\n"
msgstr "определение службы \"%s\" не найдено\n"

#: fe-connect.c:3831
#, c-format
msgid "service file \"%s\" not found\n"
msgstr "файл определений служб \"%s\" не найден\n"

#: fe-connect.c:3844
#, c-format
msgid "line %d too long in service file \"%s\"\n"
msgstr "слишком длинная строка (%d) в файле определений служб \"%s\"\n"

#: fe-connect.c:3915 fe-connect.c:3942
#, c-format
msgid "syntax error in service file \"%s\", line %d\n"
msgstr "синтаксическая ошибка в файле определения служб \"%s\" (строка %d)\n"

#: fe-connect.c:4609
msgid "connection pointer is NULL\n"
msgstr "нулевой указатель соединения\n"

#: fe-connect.c:4886
#, c-format
msgid "WARNING: password file \"%s\" is not a plain file\n"
msgstr "ВНИМАНИЕ: файл паролей \"%s\" - не обычный файл\n"

#: fe-connect.c:4895
#, c-format
msgid ""
"WARNING: password file \"%s\" has group or world access; permissions should "
"be u=rw (0600) or less\n"
msgstr ""
"ВНИМАНИЕ: к файлу паролей \"%s\" имеют доступ все или группа; права должны "
"быть u=rw (0600) или более ограниченные\n"

#: fe-connect.c:4983
#, c-format
msgid "password retrieved from file \"%s\"\n"
msgstr "пароль получен из файла \"%s\"\n"

#: fe-exec.c:810
msgid "NOTICE"
msgstr "ЗАМЕЧАНИЕ"

#: fe-exec.c:997 fe-exec.c:1054 fe-exec.c:1094
msgid "command string is a null pointer\n"
msgstr "указатель на командную строку нулевой\n"

#: fe-exec.c:1087 fe-exec.c:1182
msgid "statement name is a null pointer\n"
msgstr "указатель на имя оператора нулевой\n"

#: fe-exec.c:1102 fe-exec.c:1256 fe-exec.c:1925 fe-exec.c:2123
msgid "function requires at least protocol version 3.0\n"
msgstr "функция требует протокол минимум версии 3.0\n"

#: fe-exec.c:1213
msgid "no connection to the server\n"
msgstr "нет соединения с сервером\n"

#: fe-exec.c:1220
msgid "another command is already in progress\n"
msgstr "уже выполняется другая команда\n"

#: fe-exec.c:1332
msgid "length must be given for binary parameter\n"
msgstr "для двоичного параметра должна быть указана длина\n"

#: fe-exec.c:1585
#, c-format
msgid "unexpected asyncStatus: %d\n"
msgstr "неожиданный asyncStatus: %d\n"

#: fe-exec.c:1605
#, c-format
msgid "PGEventProc \"%s\" failed during PGEVT_RESULTCREATE event\n"
msgstr "ошибка в PGEventProc \"%s\" при обработке события PGEVT_RESULTCREATE\n"

#: fe-exec.c:1735
msgid "COPY terminated by new PQexec"
msgstr "операция COPY прервана вызовом PQexec"

#: fe-exec.c:1743
msgid "COPY IN state must be terminated first\n"
msgstr "сначала должно завершиться состояние COPY IN\n"

#: fe-exec.c:1763
msgid "COPY OUT state must be terminated first\n"
msgstr "сначала должно завершиться состояние COPY OUT\n"

#: fe-exec.c:1771
msgid "PQexec not allowed during COPY BOTH\n"
msgstr "вызов PQexec не допускается в процессе COPY BOTH\n"

#: fe-exec.c:2014 fe-exec.c:2080 fe-exec.c:2167 fe-protocol2.c:1237
#: fe-protocol3.c:1569
msgid "no COPY in progress\n"
msgstr "операция COPY не выполняется\n"

#: fe-exec.c:2359
msgid "connection in wrong state\n"
msgstr "соединение в неправильном состоянии\n"

#: fe-exec.c:2390
msgid "invalid ExecStatusType code"
msgstr "неверный код ExecStatusType"

#: fe-exec.c:2454 fe-exec.c:2477
#, c-format
msgid "column number %d is out of range 0..%d"
msgstr "номер колонки %d вне диапазона 0..%d"

#: fe-exec.c:2470
#, c-format
msgid "row number %d is out of range 0..%d"
msgstr "номер записи %d вне диапазона 0..%d"

#: fe-exec.c:2492
#, c-format
msgid "parameter number %d is out of range 0..%d"
msgstr "номер параметра %d вне диапазона 0..%d"

#: fe-exec.c:2780
#, c-format
msgid "could not interpret result from server: %s"
msgstr "не удалось интерпретировать ответ сервера: %s"

#: fe-exec.c:3019 fe-exec.c:3103
msgid "incomplete multibyte character\n"
msgstr "неполный многобайтный символ\n"

#: fe-lobj.c:152
msgid "cannot determine OID of function lo_truncate\n"
msgstr "не удалось определить OID функции lo_truncate\n"

#: fe-lobj.c:380
msgid "cannot determine OID of function lo_create\n"
msgstr "не удалось определить OID функции lo_create\n"

#: fe-lobj.c:525 fe-lobj.c:634
#, c-format
msgid "could not open file \"%s\": %s\n"
msgstr "не удалось открыть файл \"%s\": %s\n"

#: fe-lobj.c:580
#, c-format
msgid "could not read from file \"%s\": %s\n"
msgstr "не удалось прочитать файл \"%s\": %s\n"

#: fe-lobj.c:654 fe-lobj.c:678
#, c-format
msgid "could not write to file \"%s\": %s\n"
msgstr "не удалось записать файл \"%s\": %s\n"

#: fe-lobj.c:759
msgid "query to initialize large object functions did not return data\n"
msgstr "запрос инициализации функций для больших объектов не вернул данные\n"

#: fe-lobj.c:800
msgid "cannot determine OID of function lo_open\n"
msgstr "не удалось определить OID функции lo_open\n"

#: fe-lobj.c:807
msgid "cannot determine OID of function lo_close\n"
msgstr "не удалось определить OID функции lo_close\n"

#: fe-lobj.c:814
msgid "cannot determine OID of function lo_creat\n"
msgstr "не удалось определить OID функции lo_creat\n"

#: fe-lobj.c:821
msgid "cannot determine OID of function lo_unlink\n"
msgstr "не удалось определить OID функции lo_unlink\n"

#: fe-lobj.c:828
msgid "cannot determine OID of function lo_lseek\n"
msgstr "не удалось определить OID функции lo_lseek\n"

#: fe-lobj.c:835
msgid "cannot determine OID of function lo_tell\n"
msgstr "не удалось определить OID функции lo_tell\n"

#: fe-lobj.c:842
msgid "cannot determine OID of function loread\n"
msgstr "не удалось определить OID функции loread\n"

#: fe-lobj.c:849
msgid "cannot determine OID of function lowrite\n"
msgstr "не удалось определить OID функции lowrite\n"

#: fe-misc.c:270
#, c-format
msgid "integer of size %lu not supported by pqGetInt"
msgstr "функция pqGetInt не поддерживает integer размером %lu байт"

#: fe-misc.c:306
#, c-format
msgid "integer of size %lu not supported by pqPutInt"
msgstr "функция pqPutInt не поддерживает integer размером %lu байт"

#: fe-misc.c:585 fe-misc.c:784
msgid "connection not open\n"
msgstr "соединение не открыто\n"

#: fe-misc.c:711 fe-secure.c:387 fe-secure.c:466 fe-secure.c:547
#: fe-secure.c:655
msgid ""
"server closed the connection unexpectedly\n"
"\tThis probably means the server terminated abnormally\n"
"\tbefore or while processing the request.\n"
msgstr ""
"сервер неожиданно закрыл соединение\n"
"\tСкорее всего сервер прекратил работу из-за сбоя\n"
"\tдо или в процессе выполнения запроса.\n"

#: fe-misc.c:948
msgid "timeout expired\n"
msgstr "таймаут\n"

#: fe-misc.c:993
msgid "socket not open\n"
msgstr "сокет не открыт\n"

#: fe-misc.c:1016
#, c-format
msgid "select() failed: %s\n"
msgstr "ошибка в select(): %s\n"

#: fe-protocol2.c:91
#, c-format
msgid "invalid setenv state %c, probably indicative of memory corruption\n"
msgstr "неверное состояние setenv %c - возможно разрушение памяти\n"

#: fe-protocol2.c:390
#, c-format
msgid "invalid state %c, probably indicative of memory corruption\n"
msgstr "неверное состояние %c - возможно разрушение памяти\n"

#: fe-protocol2.c:479 fe-protocol3.c:186
#, c-format
msgid "message type 0x%02x arrived from server while idle"
msgstr "от сервера во время простоя получено сообщение типа 0x%02x"

#: fe-protocol2.c:522
#, c-format
msgid "unexpected character %c following empty query response (\"I\" message)"
msgstr "неожиданный символ %c вслед за пустым ответом (сообщение \"I\")"

#: fe-protocol2.c:576
msgid ""
"server sent data (\"D\" message) without prior row description (\"T\" "
"message)"
msgstr ""
"сервер отправил данные (сообщение \"D\") без предварительного описания "
"строки (сообщение \"T\")"

#: fe-protocol2.c:592
msgid ""
"server sent binary data (\"B\" message) without prior row description (\"T\" "
"message)"
msgstr ""
"сервер отправил двоичные данные (сообщение \"B\") без предварительного "
"описания строки (сообщение \"T\")"

#: fe-protocol2.c:612 fe-protocol3.c:388
#, c-format
msgid "unexpected response from server; first received character was \"%c\"\n"
msgstr "неожиданный ответ сервера; первый полученный символ: \"%c\"\n"

#: fe-protocol2.c:833 fe-protocol3.c:707
msgid "out of memory for query result\n"
msgstr "недостаточно памяти для результата запроса\n"

#: fe-protocol2.c:1280 fe-protocol3.c:1637
#, c-format
msgid "%s"
msgstr "%s"

#: fe-protocol2.c:1292
msgid "lost synchronization with server, resetting connection"
msgstr "потеряна синхронизация с сервером; попытка восстановить соединение"

#: fe-protocol2.c:1426 fe-protocol2.c:1458 fe-protocol3.c:1840
#, c-format
msgid "protocol error: id=0x%x\n"
msgstr "ошибка протокола: id=0x%x\n"

#: fe-protocol3.c:344
msgid ""
"server sent data (\"D\" message) without prior row description (\"T\" "
"message)\n"
msgstr ""
"сервер отправил данные (сообщение \"D\") без предварительного описания "
"строки (сообщение \"T\")\n"

#: fe-protocol3.c:409
#, c-format
msgid "message contents do not agree with length in message type \"%c\"\n"
msgstr "содержимое не соответствует длине в сообщении типа \"%c\"\n"

#: fe-protocol3.c:430
#, c-format
msgid "lost synchronization with server: got message type \"%c\", length %d\n"
msgstr ""
"потеряна синхронизация с сервером: получено сообщение типа \"%c\", длина %d\n"

#: fe-protocol3.c:652
msgid "unexpected field count in \"D\" message\n"
msgstr "неверное число полей в сообщении \"D\"\n"

#. translator: %s represents a digit string
#: fe-protocol3.c:798 fe-protocol3.c:817
#, c-format
msgid " at character %s"
msgstr " символ %s"

#: fe-protocol3.c:830
#, c-format
msgid "DETAIL:  %s\n"
msgstr "ПОДРОБНОСТИ:  %s\n"

#: fe-protocol3.c:833
#, c-format
msgid "HINT:  %s\n"
msgstr "ПОДСКАЗКА:  %s\n"

#: fe-protocol3.c:836
#, c-format
msgid "QUERY:  %s\n"
msgstr "ЗАПРОС:  %s\n"

#: fe-protocol3.c:839
#, c-format
msgid "CONTEXT:  %s\n"
msgstr "КОНТЕКСТ:  %s\n"

#: fe-protocol3.c:851
msgid "LOCATION:  "
msgstr "ПОЛОЖЕНИЕ:  "

#: fe-protocol3.c:853
#, c-format
msgid "%s, "
msgstr "%s, "

#: fe-protocol3.c:855
#, c-format
msgid "%s:%s"
msgstr "%s:%s"

#: fe-protocol3.c:1079
#, c-format
msgid "LINE %d: "
msgstr "СТРОКА %d: "

#: fe-protocol3.c:1465
msgid "PQgetline: not doing text COPY OUT\n"
msgstr "PQgetline можно вызывать только во время COPY OUT с текстом\n"

#: fe-secure.c:271 fe-secure.c:1126 fe-secure.c:1346
#, c-format
msgid "could not acquire mutex: %s\n"
msgstr "не удалось заблокировать семафор: %s\n"

#: fe-secure.c:283
#, c-format
msgid "could not establish SSL connection: %s\n"
msgstr "не удалось установить SSL-соединение: %s\n"

#: fe-secure.c:392 fe-secure.c:552 fe-secure.c:1464
#, c-format
msgid "SSL SYSCALL error: %s\n"
msgstr "ошибка SSL SYSCALL: %s\n"

#: fe-secure.c:399 fe-secure.c:559 fe-secure.c:1468
msgid "SSL SYSCALL error: EOF detected\n"
msgstr "ошибка SSL SYSCALL: конец файла (EOF)\n"

#: fe-secure.c:410 fe-secure.c:570 fe-secure.c:1477
#, c-format
msgid "SSL error: %s\n"
msgstr "ошибка SSL: %s\n"

#: fe-secure.c:424 fe-secure.c:584
msgid "SSL connection has been closed unexpectedly\n"
msgstr "SSL-соединение было неожиданно закрыто\n"

#: fe-secure.c:430 fe-secure.c:590 fe-secure.c:1486
#, c-format
msgid "unrecognized SSL error code: %d\n"
msgstr "нераспознанный код ошибки SSL: %d\n"

#: fe-secure.c:474
#, c-format
msgid "could not receive data from server: %s\n"
msgstr "не удалось получить данные с сервера: %s\n"

#: fe-secure.c:662
#, c-format
msgid "could not send data to server: %s\n"
msgstr "не удалось передать данные серверу: %s\n"

#: fe-secure.c:782 fe-secure.c:799
msgid "could not get server common name from server certificate\n"
msgstr "не удалось получить имя сервера из сертификата\n"

#: fe-secure.c:812
msgid "SSL certificate's common name contains embedded null\n"
msgstr "Имя SSL-сертификата включает нулевой байт\n"

#: fe-secure.c:824
msgid "host name must be specified for a verified SSL connection\n"
msgstr "для проверенного SSL-соединения требуется указать имя узла\n"

#: fe-secure.c:838
#, c-format
msgid "server common name \"%s\" does not match host name \"%s\"\n"
msgstr "имя в сертификате \"%s\" не совпадает с именем сервера \"%s\"\n"

#: fe-secure.c:973
#, c-format
msgid "could not create SSL context: %s\n"
msgstr "не удалось создать контекст SSL: %s\n"

#: fe-secure.c:1096
#, c-format
msgid "could not open certificate file \"%s\": %s\n"
msgstr "не удалось открыть файл сертификата \"%s\": %s\n"

#: fe-secure.c:1135 fe-secure.c:1150
#, c-format
msgid "could not read certificate file \"%s\": %s\n"
msgstr "не удалось прочитать файл сертификата \"%s\": %s\n"

#: fe-secure.c:1205
#, c-format
msgid "could not load SSL engine \"%s\": %s\n"
msgstr "не удалось загрузить модуль SSL ENGINE \"%s\": %s\n"

#: fe-secure.c:1217
#, c-format
msgid "could not initialize SSL engine \"%s\": %s\n"
msgstr "не удалось инициализировать модуль SSL ENGINE \"%s\": %s\n"

#: fe-secure.c:1233
#, c-format
msgid "could not read private SSL key \"%s\" from engine \"%s\": %s\n"
msgstr "не удалось прочитать закрытый ключ SSL \"%s\" из модуля \"%s\": %s\n"

#: fe-secure.c:1247
#, c-format
msgid "could not load private SSL key \"%s\" from engine \"%s\": %s\n"
msgstr "не удалось загрузить закрытый ключ SSL \"%s\" из модуля \"%s\": %s\n"

#: fe-secure.c:1284
#, c-format
msgid "certificate present, but not private key file \"%s\"\n"
msgstr "сертификат присутствует, но файла закрытого ключа \"%s\" нет\n"

#: fe-secure.c:1292
#, c-format
msgid ""
"private key file \"%s\" has group or world access; permissions should be "
"u=rw (0600) or less\n"
msgstr ""
"к файлу закрытого ключа \"%s\" имеют доступ все или группа; права должны "
"быть u=rw (0600) или более ограниченные\n"

#: fe-secure.c:1303
#, c-format
msgid "could not load private key file \"%s\": %s\n"
msgstr "не удалось загрузить файл закрытого ключа \"%s\": %s\n"

#: fe-secure.c:1317
#, c-format
msgid "certificate does not match private key file \"%s\": %s\n"
msgstr "сертификат не соответствует файлу закрытого ключа \"%s\": %s\n"

#: fe-secure.c:1355
#, c-format
msgid "could not read root certificate file \"%s\": %s\n"
msgstr "не удалось прочитать файл корневых сертификатов \"%s\": %s\n"

#: fe-secure.c:1385
#, c-format
msgid "SSL library does not support CRL certificates (file \"%s\")\n"
msgstr "Библиотека SSL не поддерживает проверку CRL (файл \"%s\")\n"

#: fe-secure.c:1418
msgid ""
"could not get home directory to locate root certificate file\n"
"Either provide the file or change sslmode to disable server certificate "
"verification.\n"
msgstr ""
"не удалось получить домашний каталог для поиска файла корневых сертификатов\n"
"Укажите полный путь к файлу или отключите проверку сертификата сервера, "
"изменив sslmode.\n"

#: fe-secure.c:1422
#, c-format
msgid ""
"root certificate file \"%s\" does not exist\n"
"Either provide the file or change sslmode to disable server certificate "
"verification.\n"
msgstr ""
"файл корневых сертификатов \"%s\" не существует\n"
"Укажите полный путь к файлу или отключите проверку сертификата сервера, "
"изменив sslmode.\n"

#: fe-secure.c:1505
#, c-format
msgid "certificate could not be obtained: %s\n"
msgstr "не удалось получить сертификат: %s\n"

#: fe-secure.c:1582
msgid "no SSL error reported"
msgstr "нет сообщения об ошибке SSL"

#: fe-secure.c:1591
#, c-format
msgid "SSL error code %lu"
msgstr "код ошибки SSL: %lu"
